src/savegame_json.cpp

```diff
@@ -688,7 +690,7 @@ void player::deserialize(JsonIn &jsin)
     } else {
         player_activity temp;
         data.read("backlog", temp);
-        backlog.push_front( temp );
+        backlog.push_front( std::move( temp ) );
```

Comment:
It's a throwback from when `player_activity` wasn't copy-constructible due to the presence of `item_location`